avaframe/com1DFA/DFAfunctionsCython.pyx

```diff
@@ -1190,34 +1190,65 @@ def computeGradC(cfg, particles, headerNeighbourGrid, headerNormalGrid, double[:
         for p in range(iPstart, iPend):
             # index of particle in neighbour box
             l = partInCell[p]
-            if j != l:
+            if k != l:
                 dx = xArray[l] - x
                 dy = yArray[l] - y
                 dz = zArray[l] - z
                 if SPHoption == 1:
-                  dz = 0
-                  # get norm of r = xj - xl
+                  # like option 2 with dz!=0
+                  #dz = 0
+                  # get norm of r = xk - xl
```

Comment:
We need to adjust a bit the options here so that sphOption 1 is what the master does and create new option where the dz is not zero ...

Reply:
okay, that's relevant :)

Reply:
in your sph options in the inifile, there isn't the description of the last option. could you describe it please  ?